wger/nutrition/views/plan.py

```diff
@@ -48,10 +50,21 @@
 
 @login_required
 def overview(request):
+    '''
+    Loads nutrition plan objects into cache and redirects to the overview page
+    '''
     template_data = {}
     template_data.update(csrf(request))
 
     plans = NutritionPlan.objects.filter(user=request.user)
+    for plan in plans:
+        cache_value = cache.get(cache_mapper.get_nutrition_plan(int(plan.id)))
+        if not cache_value:
+            cache.set(cache_mapper.get_nutrition_plan(int(plan.id)),
+                      plan.get_nutritional_values()['total']['energy'])
+            plan.cache_value = cache.get(cache_mapper.get_nutrition_plan(int(plan.id)))
+        else:
+            plan.cache_value = cache_value
     template_data['plans'] = plans
```

Comment:
seems like the looping doesn't change the data we're sending to the template. we're still using `plans`.